geopy/geocoders/osm.py

```diff
@@ -248,6 +249,58 @@ def reverse(
             self._call_geocoder(url, timeout=timeout), exactly_one
         )
 
+    def lookup(
+            self,
+            osm_ids,
+            timeout=None,
+            addressdetails=False,
+            language=False
+    ):
+        """
+        Returns address of one or multiple OSM objects.
+
+        :param osm_ids: An individual or a list of osm_ids to lookup
```

Comment:
i think it's worth mentioning that this is either a string of `OSM_TYPE + OSM_ID` or a list of such strings (in the osm docs an osm id is just the number, you need to prefix it with a type to get something you can pass to lookup)